runtime/auth/aws-signing-common/common/src/aws/smithy/kotlin/runtime/auth/awssigning/AwsSigningConfig.kt

```diff
@@ -164,6 +164,14 @@ public class AwsSigningConfig(builder: Builder) {
      */
     public val expiresAfter: Duration? = builder.expiresAfter
 
+    /**
+     * Flag enabling whether detailed trace logging is enabled (if the signer implementation supports it). When true
+     * signers should emit intermediate logging details such as the canonical request and string to sign at the trace
```

Comment:
> and string to sign

It looks the string to sign always gets logged (which I think is what we agreed to do). So should we only reference the canonical request in these docs?

Reply:
I'd go the other way, actually. The `SdkLogMode` option is named `LogSigning`. At a glance, that indicates to me that _all_ signing logging will be enabled/disabled by that flag. (The KDocs add some nuance but I think they actually _undermine_ the name.)

I think we should just make that the case: canonical request, string to sign, _and_ final signature should be gated by `LogSigning`.